peps/pep-0012.rst

```diff
@@ -647,8 +647,27 @@ If you find that you need to use a backslash in your text, consider
 using inline literals or a literal block instead.
 
 
-Canonical Documentation and Intersphinx
----------------------------------------
+Intersphinx
+-----------
+
+You can use
+`Intersphinx references
+<https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_
+to other Sphinx sites,
+currently the `Python documentation <https://docs.python.org/>`_
```

Comment:
There's currently also Intersphinx to Trio, but I don't think we need to list them all here:

```suggestion
such as the `Python documentation <https://docs.python.org/>`_
```